project.py

```diff
@@ -468,7 +471,7 @@ def find_and_draw_edges(self, image, origin):
             self.write_image(our_cnt, img, mask)
 
         self.frame_counter = (self.frame_counter + 1) % self.save_interval
-        if self.debug:
+        if self.debug is True:
```

Comment:
@Foxiy is this really necessary? Would `== True` suffice? My point is that `is` is a pretty strange in this context.